File: Semestrul 4/MPP/lab09-10/server .net/TriathlonNetworking/DTOs/RefereeDTO.cs
using System;

namespace Triathlon.Network.Dto
{
    [Serializable]
    public class RefereeDTO
    {
        private long id;
        private string username;
        private string password;

        public RefereeDTO() { }

        public RefereeDTO(long id, string username, string password)
        {
            this.id = id;
            this.username = username;
            this.password = password;
        }

        public long Id { get { return id; } set { id = value; } }
        public string Username { get { return username; } set { username = value; } }
        public string Password { get { return password; } set { password = value; } }
    }
}

File: Semestrul 4/SDI/lab09-10/client java/TriathlonNetworking/src/main/java/triathlon/network/dto/RefereeDTO.java
package triathlon.network.dto;

import java.io.Serializable;

public class RefereeDTO implements Serializable {
    private Long id;
    private String username;
    private String password;

    public RefereeDTO() {}

    public RefereeDTO(Long id, String username, String password) {
        this.id = id;
        this.username = username;
        this.password = password;
    }

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public String getUsername() { return username; }
    public void setUsername(String username) { this.username = username; }
    public String getPassword() { return password; }
    public void setPassword(String password) { this.password = password; }
}